include/integratorxx/quadratures/treutlerahlrichs.hpp

```diff
@@ -115,12 +117,45 @@ struct quadrature_traits<
         ( sqrt_term - alpha * log_term / sqrt_term );
     }
 
+    return std::make_tuple( points, weights );
 
+  }
 
-    return std::make_tuple( points, weights );
+};
+
+
+#else
+
+struct TreutlerAhlrichsM4RadialTraits {
 
+  inline static constexpr double alpha = 0.6;
```

Comment:
It doesn't seem like we have M3 quadrature, which is the simpler scheme with `alpha=0`

Reply:
Yes, now that things are modular I was planning on adding both in a later PR. We can add them here though, I have no strong opinion.

Reply:
Should `alpha=0` be a special case in the code, though?

Reply:
It will be branching logic either way (the Jacobian cares) - whether it's a separate implementation depends on how much we want to expose to the user by default. Will the typical user want to be able to manually specify `alpha` or just specify `M3`/`M4`? My guess is the latter.

Reply:
This was changed to a runtime parameter in https://github.com/wavefunction91/IntegratorXX/pull/27/commits/ffad94c7b88dba57ebf5f552938cf2c69fabfd6b and documented in https://github.com/wavefunction91/IntegratorXX/pull/27/commits/5842dbb0ef8ab2ad08bd74d756b4633c65e64bb4. I'm going to call this point resolved if its all the same to you